setup.cfg

```diff
@@ -1,12 +1,12 @@
 [flake8]
-exclude = .git,__pycache__,node_modules
+exclude = .git,__pycache__,node_modules,venv,treeherder/extract/vendor
```

Comment:
Do we still need the exclusion for `treeherder/extract/vendor` now that it's a PyPi package?

Reply:
@camd  sorry, no.  Just like `node_modules` it is a development directory:  I can not run `./runtests.sh` locally without it.